conf/config.local.php

```diff
@@ -455,7 +455,7 @@
  * Défaut : STARTTLS
  */
 
-//const SMTP_SECURITY = 'NONE';
+const SMTP_SECURITY = 'NONE';
```

Comment:
Needs to stay at @default, we do not support non-encrypted traffic :/

Are you getting any error?

Reply:
Hello, I try a lot of configurations, but still unable to send emails like this, it works only with php mail by default

last log error (with SMTP_HOST = 'localhost';

```
[15-Nov-2023 23:45:45 Europe/Paris] =========== Error ref. 1z9qjp80 ===========

KD2\SMTP_Exception: SMTP AUTH error: 503 5.5.1 Error: authentication not enabled
 in /var/www/paheko/include/lib/KD2/SMTP.php:190
Stack trace:
#0 /var/www/paheko/include/lib/KD2/SMTP.php(224): KD2\SMTP->authenticate()
#1 /var/www/paheko/include/lib/KD2/SMTP.php(408): KD2\SMTP->rawSend('...', Array, '...')
#2 /var/www/paheko/include/lib/Paheko/Email/Emails.php(671): KD2\SMTP->send(Object(KD2\Mail_Message))
#3 /var/www/paheko/include/lib/Paheko/Email/Emails.php(650): Paheko\Email\Emails::sendMessage(2, Object(KD2\Mail_Message))
#4 /var/www/paheko/include/lib/Paheko/Email/Emails.php(371): Paheko\Email\Emails::send(2, '...', Array, '...', '...', NULL, Array)
#5 /var/www/paheko/scripts/emails.php(16): Paheko\Email\Emails::runQueue()
#6 {main}

<errorReport>
{
    "errors": [
        {
            "message": "SMTP AUTH error: 503 5.5.1 Error: authentication not enabled\r\n",
            "errorCode": 0,
            "type": "KD2\\SMTP_Exception",
            "backtrace": [
                {
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 190,
                    "code": {
                        "186": "\t\t\t$this->_write('AUTH LOGIN');",
                        "187": "",
                        "188": "\t\t\tif ($this->_readCode() != 334)",
                        "189": "\t\t\t{",
                        "190": "\t\t\t\tthrow new SMTP_Exception('SMTP AUTH error: '.$this->last_line);",
                        "191": "\t\t\t}",
                        "192": "",
                        "193": "\t\t\t$this->_write(base64_encode($this->username));",
                        "194": ""
                    }
                },
                {
                    "function": "KD2\\SMTP->authenticate",
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 224,
                    "code": {
                        "220": "\t{",
                        "221": "\t\tif (is_null($this->conn))",
                        "222": "\t\t{",
                        "223": "\t\t\t$this->connect();",
                        "224": "\t\t\t$this->authenticate();",
                        "225": "\t\t}",
                        "226": "",
                        "227": "\t\t$this->_write('RSET');",
                        "228": ""
                    }
                },
                {
                    "function": "KD2\\SMTP->rawSend",
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 408,
                    "args": {
                        "$from": "string(24) \"contact@ecranvillage.net\"",
                        "$to": "array(1) {\n  [0]=> string(19) \"rodolphe@linux07.fr\"\n  }",
                        "$data": "string(16169) \"From: \"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\r\nTo: rodolphe@linux07.fr\r\nSubject: Essai nouvel 15 nov 23:44\r\nReply-To: rodinux07@rodinux.fr\r\nMessage-Id: <65554a19372a9.3iyudsavaq@wkcwocsk.kok>\r\nList-Unsubscribe: <https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\r\nList-Unsubscribe-Post: Unsubscribe=Yes\r\nReturn-Path: contact@ecranvillage.net\r\nX-Auto-Response-Suppress: All\r\nDate: Wed, 15 Nov 2023 23:45:45 +0100\r\nContent-Type: multipart\/mixed; boundary=\"==_=_65554a19371da-783d6703b7\"\r\nContent-Transfer-Encoding: 8bit\r\nMime-Version: 1.0\r\n\r\nThis is a message in multipart MIME format. \r\nYour mail client should not be displaying this. \r\nConsider upgrading your mail client to view this message correctly.\r\n\r\n--==_=_65554a19371da-783d6703b7\r\nContent-Type: multipart\/alternative; boundary=\"alt=_-===_=_65554a19371da-783d6703b7\"\r\n\r\n\r\n--alt=_-===_=_65554a19371da-783d6703b7\r\nContent-Type: text\/plain; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\nnouvel essai !!\r\n\r\net oui on tente !!=20\r\n\r\n--=20\r\nEcran Village\r\n\r\nVous recevez ce message car vous =C3=AAtes dans nos contacts.\r\nPour ne plus jamais recevoir de message de notre part cliquez ici :\r\n\r\nhttps:\/\/paheko.rodinux.fr\/?un=3DeJEHFwsd9mkd9gcnrYK5G0Yglrw\r\n\r\n--alt=_-===_=_65554a19371da-783d6703b7\r\nContent-Type: text\/html; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\n<!DOCTYPE html>\r\n<html>\r\n<head>\r\n<style type=3D\"text\/css\">\r\n=09* { margin: 0; padding: 0; }\r\n=09body {\r\n=09=09background: #fff;\r\n=09=09color: #000;\r\n=09=09font-family: sans-serif;\r\n=09}\r\n=09main {\r\n=09=09background: rgba(200, 200, 200, 0.2);\r\n=09=09padding: 10px;\r\n=09=09padding-bottom: 50px;\r\n=09}\r\n=09a {\r\n=09=09color: #009;\r\n=09=09text-decoration: underline;\r\n=09}\r\n=09.web-content, .footer {\r\n=09=09padding: 10px;\r\n=09=09background: #fff;\r\n=09=09max-width: 750px;\r\n=09=09margin: 10px auto;\r\n=09=09border-radius: 5px;\r\n=09}\r\n=09.footer {\r\n=09=09background: rgba(100, 100, 100, 0.5);\r\n=09}\r\n=09.footer h3, .foot\u2026"
                    },
                    "code": {
                        "404": "\t\t\textract($msg);",
                        "405": "\t\t}",
                        "406": "",
                        "407": "\t\t\/\/ Send email",
                        "408": "\t\treturn $this->rawSend($from, $to, $message);",
                        "409": "\t}",
                        "410": "",
                        "411": "\t\/**",
                        "412": "\t * Takes a string like a From, Cc, To or Bcc header and gets out all the email"
                    }
                },
                {
                    "function": "KD2\\SMTP->send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 671,
                    "args": {
                        "$r": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(25) \"Essai nouvel 15 nov 23:44\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(39) \"<65554a19372a9.3iyudsavaq@wkcwocsk.kok>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(243) \"nouvel essai !!\r\n\r\net oui on tente !! \n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13793) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisat\u2026"
                    },
                    "code": {
                        "667": "\t\t\t$secure = constant($const);",
                        "668": "",
                        "669": "\t\t\t$smtp = new SMTP(SMTP_HOST, SMTP_PORT, SMTP_USER, SMTP_PASSWORD, $secure, SMTP_HELO_HOSTNAME);",
                        "670": "",
                        "671": "\t\t\t$smtp->send($message);",
                        "672": "\t\t}",
                        "673": "\t\telse {",
                        "674": "\t\t\t$message->send();",
                        "675": "\t\t}"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::sendMessage",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 650,
                    "args": {
                        "$context": "int(2)",
                        "$message": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(25) \"Essai nouvel 15 nov 23:44\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(39) \"<65554a19372a9.3iyudsavaq@wkcwocsk.kok>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(243) \"nouvel essai !!\r\n\r\net oui on tente !! \n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13793) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisat\u2026"
                    },
                    "code": {
                        "646": "\t\tif ($pgp_key && $can_use_encryption) {",
                        "647": "\t\t\t$message->encrypt($pgp_key);",
                        "648": "\t\t}",
                        "649": "",
                        "650": "\t\tself::sendMessage($context, $message);",
                        "651": "\t}",
                        "652": "",
                        "653": "\tstatic public function sendMessage(int $context, Mail_Message $message): void",
                        "654": "\t{"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 371,
                    "args": {
                        "$context": "int(2)",
                        "$recipient_hash": "string(40) \"789107170b1df6691df60727ad82b91b462096bc\"",
                        "$headers": "array(3) {\n  [\"From\"]=> string(48) \"\"Administrateur Garradin\" <rodinux07@rodinux.fr>\"\n  [\"To\"]=> string(19) \"rodolphe@linux07.fr\"\n  [\"Subject\"]=> string(25) \"Essai nouvel 15 nov 23:44\"\n  }",
                        "$content": "string(243) \"nouvel essai !!\r\n\r\net oui on tente !! \n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"",
                        "$content_html": "string(13793) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisation de l'administration.\n *\n * G\u00e9n\u00e9ralement il n'est pas n\u00e9cessaire de le modifier.\n *\/\n\n.protected-contact::before {\n    content: attr(data-a) \"\\0040\" attr(data-b) \".\" attr(data-c);\n}\n\n.web-content {\n    overflow-wrap: break-word;\n    word-wrap: break-word;\n\n    word-break: break-word;\n\n    -ms-hyphens: auto;\n    -moz-hyphens: auto;\n    -webkit-hyphens: auto;\n    hyphens: auto;\n}\n\n.web-content figure {\n    margin: 0;\n    padding: 0;\n}\n\n.web-content p, .web-content h1, .web-content h2, .web-content h3, .web-content h4, .web-content h5, .web-content h6,\n.web-content ul, .web-content ol, .web-content table, .web-content blockquote, .web-content pre {\n    margin-bottom: 1rem;\n}\n\n.web-content ul, .web-content ol, .web-content dd {\n    margin-left: 2em;\n}\n\n.web-content ul {\n    list-style-type: disc;\n}\n\n.web-content ol {\n    list-style-type: decimal;\n}\n\n.web-content blockquote::before {\n    content: \"\u201d\";\n    color: rgba(0, 0, 0, 0.5);\n    display: block;\n    position: absolute;\n    font-style: italic;\n    font-size: 3rem;\n    line-height: 2rem;\n    margin-left: -3rem;\n}\n\n.web-content blockquote {\n    font-size: 1.1em;\n    padding-left: 3rem;\n}\n\n.web-content a.footnote-ref {\n    vertical-align: super;\n    font-size: small;\n}\n\n.web-content a\u2026",
                        "$pgp_key": "NULL",
                        "$attachments": "array(0) {\n  }"
                    },
                    "code": {
                        "367": "",
                        "368": "\t\t\ttry {",
                        "369": "\t\t\t\t$attachments = $db->getAssoc('SELECT id, path FROM emails_queue_attachments WHERE id_queue = ?;', $row->id);",
                        "370": "\t\t\t\t$all_attachments = array_merge($all_attachments, $attachments);",
                        "371": "\t\t\t\tself::send($row->context, $row->recipient_hash, $headers, $row->content, $row->content_html, $row->recipient_pgp_key, $attachments);",
                        "372": "\t\t\t}",
                        "373": "\t\t\tcatch (\\Exception $e) {",
                        "374": "\t\t\t\t\/\/ If sending fails, at least save what has been sent so far",
                        "375": "\t\t\t\t\/\/ so they won't get re-sent again"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::runQueue",
                    "file": "...\/scripts\/emails.php",
                    "line": 16,
                    "code": {
                        "12": "",
                        "13": "$quiet = !empty($_SERVER['argv'][1]) && $_SERVER['argv'][1] == '-q';",
                        "14": "",
                        "15": "\/\/ Send messages in queue",
                        "16": "$sent = Emails::runQueue();",
                        "17": "",
                        "18": "$count = Emails::countQueue();",
                        "19": "",
                        "20": "if (!$quiet) {"
                    }
                }
            ]
        }
    ],
    "context": {
        "date": "2023-11-15T23:45:45+01:00",
        "duration": 58.290958404541016,
        "environment": "development",
        "http_files": "array(0) {\n  }",
        "http_method": null,
        "http_post": "array(0) {\n  }",
        "id": "1z9qjp80",
        "language": "PHP 8.2.12",
        "memory_peak": 4194304,
        "memory_used": 4194304,
        "os": "Linux",
        "paheko_data_root": "\/home\/yunohost.app\/paheko\/data",
        "paheko_version": "1.3.2",
        "php_sapi": "cli",
        "remote_ip": null,
        "root_directory": "\/var\/www\/paheko"
    }
}
</errorReport>
```

Reply:
and here a repport whith SMTP_HOST = '__DOMAIN__';

```
</errorReport>

[16-Nov-2023 00:10:01 Europe/Paris] =========== Error ref. 7f5kmimv ===========

KD2\SMTP_Exception: SMTP AUTH error: 503 5.5.1 Error: authentication not enabled
 in /var/www/paheko/include/lib/KD2/SMTP.php:190
Stack trace:
#0 /var/www/paheko/include/lib/KD2/SMTP.php(224): KD2\SMTP->authenticate()
#1 /var/www/paheko/include/lib/KD2/SMTP.php(408): KD2\SMTP->rawSend('...', Array, '...')
#2 /var/www/paheko/include/lib/Paheko/Email/Emails.php(671): KD2\SMTP->send(Object(KD2\Mail_Message))
#3 /var/www/paheko/include/lib/Paheko/Email/Emails.php(650): Paheko\Email\Emails::sendMessage(2, Object(KD2\Mail_Message))
#4 /var/www/paheko/include/lib/Paheko/Email/Emails.php(371): Paheko\Email\Emails::send(2, '...', Array, '...', '...', NULL, Array)
#5 /var/www/paheko/scripts/emails.php(16): Paheko\Email\Emails::runQueue()
#6 {main}

<errorReport>
{
    "errors": [
        {
            "message": "SMTP AUTH error: 503 5.5.1 Error: authentication not enabled\r\n",
            "errorCode": 0,
            "type": "KD2\\SMTP_Exception",
            "backtrace": [
                {
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 190,
                    "code": {
                        "186": "\t\t\t$this->_write('AUTH LOGIN');",
                        "187": "",
                        "188": "\t\t\tif ($this->_readCode() != 334)",
                        "189": "\t\t\t{",
                        "190": "\t\t\t\tthrow new SMTP_Exception('SMTP AUTH error: '.$this->last_line);",
                        "191": "\t\t\t}",
                        "192": "",
                        "193": "\t\t\t$this->_write(base64_encode($this->username));",
                        "194": ""
                    }
                },
                {
                    "function": "KD2\\SMTP->authenticate",
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 224,
                    "code": {
                        "220": "\t{",
                        "221": "\t\tif (is_null($this->conn))",
                        "222": "\t\t{",
                        "223": "\t\t\t$this->connect();",
                        "224": "\t\t\t$this->authenticate();",
                        "225": "\t\t}",
                        "226": "",
                        "227": "\t\t$this->_write('RSET');",
                        "228": ""
                    }
                },
                {
                    "function": "KD2\\SMTP->rawSend",
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 408,
                    "args": {
                        "$from": "string(24) \"contact@ecranvillage.net\"",
                        "$to": "array(1) {\n  [0]=> string(19) \"rodolphe@linux07.fr\"\n  }",
                        "$data": "string(16171) \"From: \"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\r\nTo: rodolphe@linux07.fr\r\nSubject: Essai nouvel 156 nov 00:09\r\nReply-To: rodinux07@rodinux.fr\r\nMessage-Id: <65554fc9350d9.shj3h43sj7@cs.wog>\r\nList-Unsubscribe: <https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\r\nList-Unsubscribe-Post: Unsubscribe=Yes\r\nReturn-Path: contact@ecranvillage.net\r\nX-Auto-Response-Suppress: All\r\nDate: Thu, 16 Nov 2023 00:10:01 +0100\r\nContent-Type: multipart\/mixed; boundary=\"==_=_65554fc935003-0e73972431\"\r\nContent-Transfer-Encoding: 8bit\r\nMime-Version: 1.0\r\n\r\nThis is a message in multipart MIME format. \r\nYour mail client should not be displaying this. \r\nConsider upgrading your mail client to view this message correctly.\r\n\r\n--==_=_65554fc935003-0e73972431\r\nContent-Type: multipart\/alternative; boundary=\"alt=_-===_=_65554fc935003-0e73972431\"\r\n\r\n\r\n--alt=_-===_=_65554fc935003-0e73972431\r\nContent-Type: text\/plain; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\nnouvel essai=20\r\n\r\nh=C3=A9 oui, on y crois !!!\r\n\r\n--=20\r\nEcran Village\r\n\r\nVous recevez ce message car vous =C3=AAtes dans nos contacts.\r\nPour ne plus jamais recevoir de message de notre part cliquez ici :\r\n\r\nhttps:\/\/paheko.rodinux.fr\/?un=3DeJEHFwsd9mkd9gcnrYK5G0Yglrw\r\n\r\n--alt=_-===_=_65554fc935003-0e73972431\r\nContent-Type: text\/html; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\n<!DOCTYPE html>\r\n<html>\r\n<head>\r\n<style type=3D\"text\/css\">\r\n=09* { margin: 0; padding: 0; }\r\n=09body {\r\n=09=09background: #fff;\r\n=09=09color: #000;\r\n=09=09font-family: sans-serif;\r\n=09}\r\n=09main {\r\n=09=09background: rgba(200, 200, 200, 0.2);\r\n=09=09padding: 10px;\r\n=09=09padding-bottom: 50px;\r\n=09}\r\n=09a {\r\n=09=09color: #009;\r\n=09=09text-decoration: underline;\r\n=09}\r\n=09.web-content, .footer {\r\n=09=09padding: 10px;\r\n=09=09background: #fff;\r\n=09=09max-width: 750px;\r\n=09=09margin: 10px auto;\r\n=09=09border-radius: 5px;\r\n=09}\r\n=09.footer {\r\n=09=09background: rgba(100, 100, 100, 0.5);\r\n=09}\r\n=09.footer h3, .foo\u2026"
                    },
                    "code": {
                        "404": "\t\t\textract($msg);",
                        "405": "\t\t}",
                        "406": "",
                        "407": "\t\t\/\/ Send email",
                        "408": "\t\treturn $this->rawSend($from, $to, $message);",
                        "409": "\t}",
                        "410": "",
                        "411": "\t\/**",
                        "412": "\t * Takes a string like a From, Cc, To or Bcc header and gets out all the email"
                    }
                },
                {
                    "function": "KD2\\SMTP->send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 671,
                    "args": {
                        "$r": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(26) \"Essai nouvel 156 nov 00:09\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(33) \"<65554fc9350d9.shj3h43sj7@cs.wog>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(245) \"nouvel essai \r\n\r\nh\u00e9 oui, on y crois !!!\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13794) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisation\u2026"
                    },
                    "code": {
                        "667": "\t\t\t$secure = constant($const);",
                        "668": "",
                        "669": "\t\t\t$smtp = new SMTP(SMTP_HOST, SMTP_PORT, SMTP_USER, SMTP_PASSWORD, $secure, SMTP_HELO_HOSTNAME);",
                        "670": "",
                        "671": "\t\t\t$smtp->send($message);",
                        "672": "\t\t}",
                        "673": "\t\telse {",
                        "674": "\t\t\t$message->send();",
                        "675": "\t\t}"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::sendMessage",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 650,
                    "args": {
                        "$context": "int(2)",
                        "$message": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(26) \"Essai nouvel 156 nov 00:09\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(33) \"<65554fc9350d9.shj3h43sj7@cs.wog>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(245) \"nouvel essai \r\n\r\nh\u00e9 oui, on y crois !!!\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13794) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisation\u2026"
                    },
                    "code": {
                        "646": "\t\tif ($pgp_key && $can_use_encryption) {",
                        "647": "\t\t\t$message->encrypt($pgp_key);",
                        "648": "\t\t}",
                        "649": "",
                        "650": "\t\tself::sendMessage($context, $message);",
                        "651": "\t}",
                        "652": "",
                        "653": "\tstatic public function sendMessage(int $context, Mail_Message $message): void",
                        "654": "\t{"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 371,
                    "args": {
                        "$context": "int(2)",
                        "$recipient_hash": "string(40) \"789107170b1df6691df60727ad82b91b462096bc\"",
                        "$headers": "array(3) {\n  [\"From\"]=> string(48) \"\"Administrateur Garradin\" <rodinux07@rodinux.fr>\"\n  [\"To\"]=> string(19) \"rodolphe@linux07.fr\"\n  [\"Subject\"]=> string(26) \"Essai nouvel 156 nov 00:09\"\n  }",
                        "$content": "string(245) \"nouvel essai \r\n\r\nh\u00e9 oui, on y crois !!!\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"",
                        "$content_html": "string(13794) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisation de l'administration.\n *\n * G\u00e9n\u00e9ralement il n'est pas n\u00e9cessaire de le modifier.\n *\/\n\n.protected-contact::before {\n    content: attr(data-a) \"\\0040\" attr(data-b) \".\" attr(data-c);\n}\n\n.web-content {\n    overflow-wrap: break-word;\n    word-wrap: break-word;\n\n    word-break: break-word;\n\n    -ms-hyphens: auto;\n    -moz-hyphens: auto;\n    -webkit-hyphens: auto;\n    hyphens: auto;\n}\n\n.web-content figure {\n    margin: 0;\n    padding: 0;\n}\n\n.web-content p, .web-content h1, .web-content h2, .web-content h3, .web-content h4, .web-content h5, .web-content h6,\n.web-content ul, .web-content ol, .web-content table, .web-content blockquote, .web-content pre {\n    margin-bottom: 1rem;\n}\n\n.web-content ul, .web-content ol, .web-content dd {\n    margin-left: 2em;\n}\n\n.web-content ul {\n    list-style-type: disc;\n}\n\n.web-content ol {\n    list-style-type: decimal;\n}\n\n.web-content blockquote::before {\n    content: \"\u201d\";\n    color: rgba(0, 0, 0, 0.5);\n    display: block;\n    position: absolute;\n    font-style: italic;\n    font-size: 3rem;\n    line-height: 2rem;\n    margin-left: -3rem;\n}\n\n.web-content blockquote {\n    font-size: 1.1em;\n    padding-left: 3rem;\n}\n\n.web-content a.footnote-ref {\n    vertical-align: super;\n    font-size: small;\n}\n\n.web-content a\u2026",
                        "$pgp_key": "NULL",
                        "$attachments": "array(0) {\n  }"
                    },
                    "code": {
                        "367": "",
                        "368": "\t\t\ttry {",
                        "369": "\t\t\t\t$attachments = $db->getAssoc('SELECT id, path FROM emails_queue_attachments WHERE id_queue = ?;', $row->id);",
                        "370": "\t\t\t\t$all_attachments = array_merge($all_attachments, $attachments);",
                        "371": "\t\t\t\tself::send($row->context, $row->recipient_hash, $headers, $row->content, $row->content_html, $row->recipient_pgp_key, $attachments);",
                        "372": "\t\t\t}",
                        "373": "\t\t\tcatch (\\Exception $e) {",
                        "374": "\t\t\t\t\/\/ If sending fails, at least save what has been sent so far",
                        "375": "\t\t\t\t\/\/ so they won't get re-sent again"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::runQueue",
                    "file": "...\/scripts\/emails.php",
                    "line": 16,
                    "code": {
                        "12": "",
                        "13": "$quiet = !empty($_SERVER['argv'][1]) && $_SERVER['argv'][1] == '-q';",
                        "14": "",
                        "15": "\/\/ Send messages in queue",
                        "16": "$sent = Emails::runQueue();",
                        "17": "",
                        "18": "$count = Emails::countQueue();",
                        "19": "",
                        "20": "if (!$quiet) {"
                    }
                }
            ]
        }
    ],
    "context": {
        "date": "2023-11-16T00:10:01+01:00",
        "duration": 54.344892501831055,
        "environment": "development",
        "http_files": "array(0) {\n  }",
        "http_method": null,
        "http_post": "array(0) {\n  }",
        "id": "7f5kmimv",
        "language": "PHP 8.2.12",
        "memory_peak": 4194304,
        "memory_used": 4194304,
        "os": "Linux",
        "paheko_data_root": "\/home\/yunohost.app\/paheko\/data",
        "paheko_version": "1.3.2",
        "php_sapi": "cli",
        "remote_ip": null,
        "root_directory": "\/var\/www\/paheko"
    }
}
</errorReport>
```

Reply:
I feel like `SMTP AUTH error: 503 5.5.1 Error: authentication not enabled` you're getting in both cases means you do password auth before server expecting it, likely before secure session starts. Have you tried changing `SECURITY` to `STARTTLS`?

Reply:
I have also try and also with `SMTP_PORT = 587`... I try this again, perhaps only with `const SMTP_SECURITY = 'STARTTLS';` first...

Reply:
This will change the error, because by default the user eamil from is the admin user of the app I think:

```
<errorReport>
{
    "errors": [
        {
            "message": "SMTP RCPT TO error: 553 5.7.1 <contact@ecranvillage.net>: Sender address rejected: not owned by user paheko\r\n",
            "errorCode": 0,
            "type": "KD2\\SMTP_Exception",
            "backtrace": [
                {
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 256,
                    "code": {
                        "252": "\t\t\t$code = $this->_readCode();",
                        "253": "",
                        "254": "\t\t\tif ($code != 250 && $code != 251)",
                        "255": "\t\t\t{",
                        "256": "\t\t\t\tthrow new SMTP_Exception('SMTP RCPT TO error: '.$this->last_line);",
                        "257": "\t\t\t}",
                        "258": "\t\t}",
                        "259": "",
                        "260": "\t\t$data = rtrim($data) . self::EOL;"
                    }
                },
                {
                    "function": "KD2\\SMTP->rawSend",
                    "file": "...\/include\/lib\/KD2\/SMTP.php",
                    "line": 408,
                    "args": {
                        "$from": "string(24) \"contact@ecranvillage.net\"",
                        "$to": "array(1) {\n  [0]=> string(19) \"rodolphe@linux07.fr\"\n  }",
                        "$data": "string(16180) \"From: \"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\r\nTo: rodolphe@linux07.fr\r\nSubject: essai du jeudi 16 nov 13:52\r\nReply-To: rodinux07@rodinux.fr\r\nMessage-Id: <655610ad74ef9.2bn1brla77@kgokgwwoso.wwc>\r\nList-Unsubscribe: <https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\r\nList-Unsubscribe-Post: Unsubscribe=Yes\r\nReturn-Path: contact@ecranvillage.net\r\nX-Auto-Response-Suppress: All\r\nDate: Thu, 16 Nov 2023 13:53:01 +0100\r\nContent-Type: multipart\/mixed; boundary=\"==_=_655610ad74e2c-971955ba63\"\r\nContent-Transfer-Encoding: 8bit\r\nMime-Version: 1.0\r\n\r\nThis is a message in multipart MIME format. \r\nYour mail client should not be displaying this. \r\nConsider upgrading your mail client to view this message correctly.\r\n\r\n--==_=_655610ad74e2c-971955ba63\r\nContent-Type: multipart\/alternative; boundary=\"alt=_-===_=_655610ad74e2c-971955ba63\"\r\n\r\n\r\n--alt=_-===_=_655610ad74e2c-971955ba63\r\nContent-Type: text\/plain; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\nessai avec smtp_security STARTTLS\r\n\r\npourquoi pas ?\r\n\r\n--=20\r\nEcran Village\r\n\r\nVous recevez ce message car vous =C3=AAtes dans nos contacts.\r\nPour ne plus jamais recevoir de message de notre part cliquez ici :\r\n\r\nhttps:\/\/paheko.rodinux.fr\/?un=3DeJEHFwsd9mkd9gcnrYK5G0Yglrw\r\n\r\n--alt=_-===_=_655610ad74e2c-971955ba63\r\nContent-Type: text\/html; charset=utf-8\r\nContent-Transfer-Encoding: quoted-printable\r\n\r\n<!DOCTYPE html>\r\n<html>\r\n<head>\r\n<style type=3D\"text\/css\">\r\n=09* { margin: 0; padding: 0; }\r\n=09body {\r\n=09=09background: #fff;\r\n=09=09color: #000;\r\n=09=09font-family: sans-serif;\r\n=09}\r\n=09main {\r\n=09=09background: rgba(200, 200, 200, 0.2);\r\n=09=09padding: 10px;\r\n=09=09padding-bottom: 50px;\r\n=09}\r\n=09a {\r\n=09=09color: #009;\r\n=09=09text-decoration: underline;\r\n=09}\r\n=09.web-content, .footer {\r\n=09=09padding: 10px;\r\n=09=09background: #fff;\r\n=09=09max-width: 750px;\r\n=09=09margin: 10px auto;\r\n=09=09border-radius: 5px;\r\n=09}\r\n=09.footer {\r\n=09=09background: rgba(100, 100, 100, 0.5);\r\n=09}\r\n=09.f\u2026"
                    },
                    "code": {
                        "404": "\t\t\textract($msg);",
                        "405": "\t\t}",
                        "406": "",
                        "407": "\t\t\/\/ Send email",
                        "408": "\t\treturn $this->rawSend($from, $to, $message);",
                        "409": "\t}",
                        "410": "",
                        "411": "\t\/**",
                        "412": "\t * Takes a string like a From, Cc, To or Bcc header and gets out all the email"
                    }
                },
                {
                    "function": "KD2\\SMTP->send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 671,
                    "args": {
                        "$r": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(27) \"essai du jeudi 16 nov 13:52\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(41) \"<655610ad74ef9.2bn1brla77@kgokgwwoso.wwc>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(256) \"essai avec smtp_security STARTTLS\r\n\r\npourquoi pas ?\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13801) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans \u2026"
                    },
                    "code": {
                        "667": "\t\t\t$secure = constant($const);",
                        "668": "",
                        "669": "\t\t\t$smtp = new SMTP(SMTP_HOST, SMTP_PORT, SMTP_USER, SMTP_PASSWORD, $secure, SMTP_HELO_HOSTNAME);",
                        "670": "",
                        "671": "\t\t\t$smtp->send($message);",
                        "672": "\t\t}",
                        "673": "\t\telse {",
                        "674": "\t\t\t$message->send();",
                        "675": "\t\t}"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::sendMessage",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 650,
                    "args": {
                        "$context": "int(2)",
                        "$message": "object(KD2\\Mail_Message) (5) {\n  [\"\u0000*\u0000headers\"]=> array(9) {\n      [\"from\"]=> string(52) \"\"Administrateur Garradin\" <paheko@paheko.rodinux.fr>\"\n      [\"to\"]=> string(19) \"rodolphe@linux07.fr\"\n      [\"subject\"]=> string(27) \"essai du jeudi 16 nov 13:52\"\n      [\"reply-to\"]=> string(20) \"rodinux07@rodinux.fr\"\n      [\"message-id\"]=> string(41) \"<655610ad74ef9.2bn1brla77@kgokgwwoso.wwc>\"\n      [\"list-unsubscribe\"]=> string(59) \"<https:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw>\"\n      [\"list-unsubscribe-post\"]=> string(15) \"Unsubscribe=Yes\"\n      [\"return-path\"]=> string(24) \"contact@ecranvillage.net\"\n      [\"x-auto-response-suppress\"]=> string(3) \"All\"\n      }\n  [\"\u0000*\u0000raw\"]=> string(0) \"\"\n  [\"\u0000*\u0000parts\"]=> array(2) {\n      [0]=> array(5) {\n          [\"type\"]=> string(10) \"text\/plain\"\n          [\"content\"]=> string(256) \"essai avec smtp_security STARTTLS\r\n\r\npourquoi pas ?\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"\n          [\"name\"]=> NULL\n          [\"cid\"]=> NULL\n          [\"encoding\"]=> NULL\n          }\n      [1]=> array(5) {\n          [\"type\"]=> string(9) \"text\/html\"\n          [\"content\"]=> string(13801) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans \u2026"
                    },
                    "code": {
                        "646": "\t\tif ($pgp_key && $can_use_encryption) {",
                        "647": "\t\t\t$message->encrypt($pgp_key);",
                        "648": "\t\t}",
                        "649": "",
                        "650": "\t\tself::sendMessage($context, $message);",
                        "651": "\t}",
                        "652": "",
                        "653": "\tstatic public function sendMessage(int $context, Mail_Message $message): void",
                        "654": "\t{"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::send",
                    "file": "...\/include\/lib\/Paheko\/Email\/Emails.php",
                    "line": 371,
                    "args": {
                        "$context": "int(2)",
                        "$recipient_hash": "string(40) \"789107170b1df6691df60727ad82b91b462096bc\"",
                        "$headers": "array(3) {\n  [\"From\"]=> string(48) \"\"Administrateur Garradin\" <rodinux07@rodinux.fr>\"\n  [\"To\"]=> string(19) \"rodolphe@linux07.fr\"\n  [\"Subject\"]=> string(27) \"essai du jeudi 16 nov 13:52\"\n  }",
                        "$content": "string(256) \"essai avec smtp_security STARTTLS\r\n\r\npourquoi pas ?\n\n-- \nEcran Village\n\nVous recevez ce message car vous \u00eates dans nos contacts.\nPour ne plus jamais recevoir de message de notre part cliquez ici :\n\nhttps:\/\/paheko.rodinux.fr\/?un=eJEHFwsd9mkd9gcnrYK5G0Yglrw\"",
                        "$content_html": "string(13801) \"\n<!DOCTYPE html>\n<html>\n<head>\n<style type=\"text\/css\">\n\t* { margin: 0; padding: 0; }\n\tbody {\n\t\tbackground: #fff;\n\t\tcolor: #000;\n\t\tfont-family: sans-serif;\n\t}\n\tmain {\n\t\tbackground: rgba(200, 200, 200, 0.2);\n\t\tpadding: 10px;\n\t\tpadding-bottom: 50px;\n\t}\n\ta {\n\t\tcolor: #009;\n\t\ttext-decoration: underline;\n\t}\n\t.web-content, .footer {\n\t\tpadding: 10px;\n\t\tbackground: #fff;\n\t\tmax-width: 750px;\n\t\tmargin: 10px auto;\n\t\tborder-radius: 5px;\n\t}\n\t.footer {\n\t\tbackground: rgba(100, 100, 100, 0.5);\n\t}\n\t.footer h3, .footer h4, .footer h5 {\n\t\ttext-align: center;\n\t\tmargin: 5px 0;\n\t}\n\t\n\t\/**\n * Ce fichier contient les styles CSS qui s'appliquent au contenu des articles et cat\u00e9gorie,\n * que ce soit sur le site public ou dans la pr\u00e9visualisation de l'administration.\n *\n * G\u00e9n\u00e9ralement il n'est pas n\u00e9cessaire de le modifier.\n *\/\n\n.protected-contact::before {\n    content: attr(data-a) \"\\0040\" attr(data-b) \".\" attr(data-c);\n}\n\n.web-content {\n    overflow-wrap: break-word;\n    word-wrap: break-word;\n\n    word-break: break-word;\n\n    -ms-hyphens: auto;\n    -moz-hyphens: auto;\n    -webkit-hyphens: auto;\n    hyphens: auto;\n}\n\n.web-content figure {\n    margin: 0;\n    padding: 0;\n}\n\n.web-content p, .web-content h1, .web-content h2, .web-content h3, .web-content h4, .web-content h5, .web-content h6,\n.web-content ul, .web-content ol, .web-content table, .web-content blockquote, .web-content pre {\n    margin-bottom: 1rem;\n}\n\n.web-content ul, .web-content ol, .web-content dd {\n    margin-left: 2em;\n}\n\n.web-content ul {\n    list-style-type: disc;\n}\n\n.web-content ol {\n    list-style-type: decimal;\n}\n\n.web-content blockquote::before {\n    content: \"\u201d\";\n    color: rgba(0, 0, 0, 0.5);\n    display: block;\n    position: absolute;\n    font-style: italic;\n    font-size: 3rem;\n    line-height: 2rem;\n    margin-left: -3rem;\n}\n\n.web-content blockquote {\n    font-size: 1.1em;\n    padding-left: 3rem;\n}\n\n.web-content a.footnote-ref {\n    vertical-align: super;\n    font-size: small;\n}\n\n.web-content a\u2026",
                        "$pgp_key": "NULL",
                        "$attachments": "array(0) {\n  }"
                    },
                    "code": {
                        "367": "",
                        "368": "\t\t\ttry {",
                        "369": "\t\t\t\t$attachments = $db->getAssoc('SELECT id, path FROM emails_queue_attachments WHERE id_queue = ?;', $row->id);",
                        "370": "\t\t\t\t$all_attachments = array_merge($all_attachments, $attachments);",
                        "371": "\t\t\t\tself::send($row->context, $row->recipient_hash, $headers, $row->content, $row->content_html, $row->recipient_pgp_key, $attachments);",
                        "372": "\t\t\t}",
                        "373": "\t\t\tcatch (\\Exception $e) {",
                        "374": "\t\t\t\t\/\/ If sending fails, at least save what has been sent so far",
                        "375": "\t\t\t\t\/\/ so they won't get re-sent again"
                    }
                },
                {
                    "function": "Paheko\\Email\\Emails::runQueue",
                    "file": "...\/scripts\/emails.php",
                    "line": 16,
                    "code": {
                        "12": "",
                        "13": "$quiet = !empty($_SERVER['argv'][1]) && $_SERVER['argv'][1] == '-q';",
                        "14": "",
                        "15": "\/\/ Send messages in queue",
                        "16": "$sent = Emails::runQueue();",
                        "17": "",
                        "18": "$count = Emails::countQueue();",
                        "19": "",
                        "20": "if (!$quiet) {"
                    }
                }
            ]
        }
    ],
    "context": {
        "date": "2023-11-16T13:53:01+01:00",
        "duration": 100.1741886138916,
        "environment": "development",
        "http_files": "array(0) {\n  }",
        "http_method": null,
        "http_post": "array(0) {\n  }",
        "id": "2fmb8nfh",
        "language": "PHP 8.2.12",
        "memory_peak": 4194304,
        "memory_used": 4194304,
        "os": "Linux",
        "paheko_data_root": "\/home\/yunohost.app\/paheko\/data",
        "paheko_version": "1.3.2",
        "php_sapi": "cli",
        "remote_ip": null,
        "root_directory": "\/var\/www\/paheko"
    }
}
</errorReport>
```

Reply:
Perhaps I need change the email of the admin user with `__APP__@__DOMAIN__` in this case ??

Reply:
Ok, I found this, if I put the correct sender user `__APP__@__DOMAIN__` in the configuration of the association, it works... but this parameters are done only after the post-install and usually the users will put another email from their association... 

for one mail, I have done a `postqueue -f` to send it, here logs mail.info

```
Nov 16 03:29:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:29:01 rodinux postfix/qmgr[1808]: A37542C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:29:01 rodinux postfix/pipe[110467]: A37542C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:29:06 rodinux postfix/pipe[111135]: A897D2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:30:01 rodinux postfix/pickup[111544]: B76292C12CA: uid=987 from=<paheko>
Nov 16 03:30:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:30:01 rodinux postfix/qmgr[1808]: B76292C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:30:01 rodinux postfix/pipe[110467]: B76292C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:30:06 rodinux postfix/pipe[111135]: BBA4B2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:31:01 rodinux postfix/pickup[111544]: 19DE82C12CA: uid=987 from=<paheko>
Nov 16 03:31:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:31:01 rodinux postfix/qmgr[1808]: 19DE82C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:31:01 rodinux postfix/pipe[110467]: 19DE82C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:31:06 rodinux postfix/pipe[111135]: 1E0C62C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:32:01 rodinux postfix/pickup[111544]: 2E2222C12CA: uid=987 from=<paheko>
Nov 16 03:32:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:32:01 rodinux postfix/qmgr[1808]: 2E2222C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:32:01 rodinux postfix/pipe[110467]: 2E2222C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:32:06 rodinux postfix/pipe[111135]: 333AE2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:33:01 rodinux postfix/pickup[111544]: 436112C12CA: uid=987 from=<paheko>
Nov 16 03:33:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:33:01 rodinux postfix/qmgr[1808]: 436112C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:33:01 rodinux postfix/pipe[110467]: 436112C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:33:06 rodinux postfix/pipe[111135]: 4881E2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:34:01 rodinux postfix/pickup[111544]: 585E62C12CA: uid=987 from=<paheko>
Nov 16 03:34:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:34:01 rodinux postfix/qmgr[1808]: 585E62C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:34:01 rodinux postfix/pipe[110467]: 585E62C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:34:06 rodinux postfix/pipe[111135]: 5D74A2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:35:01 rodinux postfix/pickup[111544]: 6A48F2C12CA: uid=987 from=<paheko>
Nov 16 03:35:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:35:01 rodinux postfix/qmgr[1808]: 6A48F2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:35:01 rodinux postfix/pipe[110467]: 6A48F2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:35:06 rodinux postfix/pipe[111135]: 6E4A82C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:36:01 rodinux postfix/pickup[111544]: C80E02C12CA: uid=987 from=<paheko>
Nov 16 03:36:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:36:01 rodinux postfix/qmgr[1808]: C80E02C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:36:01 rodinux postfix/pipe[110467]: C80E02C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:36:06 rodinux postfix/pipe[111135]: CC8FE2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:37:01 rodinux postfix/pickup[111544]: D9BC82C12CA: uid=987 from=<paheko>
Nov 16 03:37:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:37:01 rodinux postfix/qmgr[1808]: D9BC82C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:37:01 rodinux postfix/pipe[110467]: D9BC82C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:37:06 rodinux postfix/pipe[111135]: DDCCB2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:38:01 rodinux postfix/pickup[111544]: EE0312C12CA: uid=987 from=<paheko>
Nov 16 03:38:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:38:01 rodinux postfix/qmgr[1808]: EE0312C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:38:01 rodinux postfix/pipe[110467]: EE0312C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:38:07 rodinux postfix/pipe[111135]: F22342C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:39:02 rodinux postfix/pickup[111544]: 0C2A52C12CA: uid=987 from=<paheko>
Nov 16 03:39:02 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:39:02 rodinux postfix/qmgr[1808]: 0C2A52C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:39:02 rodinux postfix/pipe[110467]: 0C2A52C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:39:07 rodinux postfix/pipe[111135]: 102D72C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:40:01 rodinux postfix/pickup[111544]: 1FD382C12CA: uid=987 from=<paheko>
Nov 16 03:40:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:40:01 rodinux postfix/qmgr[1808]: 1FD382C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:40:01 rodinux postfix/pipe[110467]: 1FD382C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:40:06 rodinux postfix/pipe[111135]: 23CD12C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:41:01 rodinux postfix/pickup[111544]: 867322C12CA: uid=987 from=<paheko>
Nov 16 03:41:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:41:01 rodinux postfix/qmgr[1808]: 867322C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:41:01 rodinux postfix/pipe[110467]: 867322C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:41:06 rodinux postfix/pipe[111135]: 8AAE62C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:42:01 rodinux postfix/pickup[111544]: 9AF652C12CA: uid=987 from=<paheko>
Nov 16 03:42:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:42:01 rodinux postfix/qmgr[1808]: 9AF652C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:42:01 rodinux postfix/pipe[110467]: 9AF652C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:42:06 rodinux postfix/pipe[111135]: 9F7662C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:43:01 rodinux postfix/pickup[111544]: B08F42C12CA: uid=987 from=<paheko>
Nov 16 03:43:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:43:01 rodinux postfix/qmgr[1808]: B08F42C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:43:01 rodinux postfix/pipe[110467]: B08F42C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:43:06 rodinux postfix/pipe[111135]: B5FA82C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:44:01 rodinux postfix/pickup[111544]: C66A32C12CA: uid=987 from=<paheko>
Nov 16 03:44:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:44:01 rodinux postfix/qmgr[1808]: C66A32C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:44:01 rodinux postfix/pipe[111135]: C66A32C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:44:06 rodinux postfix/pipe[111135]: CB35E2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:45:01 rodinux postfix/pickup[111544]: D94212C12CA: uid=987 from=<paheko>
Nov 16 03:45:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:45:01 rodinux postfix/qmgr[1808]: D94212C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:45:01 rodinux postfix/pipe[111135]: D94212C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:45:06 rodinux postfix/pipe[111135]: DD3722C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:46:01 rodinux postfix/pickup[111544]: 477D32C14EE: uid=987 from=<paheko>
Nov 16 03:46:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:46:01 rodinux postfix/qmgr[1808]: 477D32C14EE: from=<paheko@rodinux.fr>, size=16211, nrcpt=1 (queue active)
Nov 16 03:46:01 rodinux postfix/pickup[111544]: 4B8592C12CA: uid=987 from=<paheko>
Nov 16 03:46:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:46:01 rodinux postfix/qmgr[1808]: 4B8592C12CA: from=<paheko@rodinux.fr>, size=617, nrcpt=1 (queue active)
Nov 16 03:46:01 rodinux postfix/pipe[111135]: 4B8592C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:46:06 rodinux postfix/pipe[111135]: 4FAB62C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5.4, delays=0/5.3/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:47:01 rodinux postfix/pickup[111544]: 5FEC92C12CA: uid=987 from=<paheko>
Nov 16 03:47:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:47:01 rodinux postfix/qmgr[1808]: 5FEC92C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:47:01 rodinux postfix/pipe[111135]: 5FEC92C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:47:06 rodinux postfix/pipe[111135]: 6430E2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:48:01 rodinux postfix/pickup[111544]: 738C72C12CA: uid=987 from=<paheko>
Nov 16 03:48:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:48:01 rodinux postfix/qmgr[1808]: 738C72C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:48:01 rodinux postfix/pipe[111135]: 738C72C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:48:06 rodinux postfix/pipe[111135]: 798622C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:49:01 rodinux postfix/pickup[111544]: 887542C12CA: uid=987 from=<paheko>
Nov 16 03:49:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:49:01 rodinux postfix/qmgr[1808]: 887542C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:49:01 rodinux postfix/pipe[111135]: 887542C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:49:06 rodinux postfix/pipe[111135]: 8D2F32C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:50:01 rodinux postfix/pickup[111544]: 9BD832C12CA: uid=987 from=<paheko>
Nov 16 03:50:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:50:01 rodinux postfix/qmgr[1808]: 9BD832C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:50:01 rodinux postfix/pipe[111135]: 9BD832C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:50:06 rodinux postfix/pipe[111135]: 9FE262C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:51:02 rodinux postfix/pickup[111544]: 01F5B2C12CA: uid=987 from=<paheko>
Nov 16 03:51:02 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:51:02 rodinux postfix/qmgr[1808]: 01F5B2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:51:02 rodinux postfix/pipe[111135]: 01F5B2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:51:07 rodinux postfix/pipe[111135]: 0617A2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:52:01 rodinux postfix/pickup[111544]: 16D5F2C12CA: uid=987 from=<paheko>
Nov 16 03:52:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:52:01 rodinux postfix/qmgr[1808]: 16D5F2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:52:01 rodinux postfix/pipe[111135]: 16D5F2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:52:06 rodinux postfix/pipe[111135]: 1B1582C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:53:01 rodinux postfix/pickup[111544]: 2E1802C12CA: uid=987 from=<paheko>
Nov 16 03:53:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:53:01 rodinux postfix/qmgr[1808]: 2E1802C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:53:01 rodinux postfix/pipe[111135]: 2E1802C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.04, delays=0.02/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:53:06 rodinux postfix/pipe[111135]: 33BBE2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:54:01 rodinux postfix/pickup[111544]: 41F672C12CA: uid=987 from=<paheko>
Nov 16 03:54:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:54:01 rodinux postfix/qmgr[1808]: 41F672C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:54:01 rodinux postfix/pipe[111135]: 41F672C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.04, delays=0.02/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:54:06 rodinux postfix/pipe[111135]: 49AAE2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:55:01 rodinux postfix/pickup[111544]: 53E342C12CA: uid=987 from=<paheko>
Nov 16 03:55:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:55:01 rodinux postfix/qmgr[1808]: 53E342C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:55:01 rodinux postfix/pipe[111135]: 53E342C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:55:06 rodinux postfix/pipe[111135]: 57E892C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:56:01 rodinux postfix/pickup[111544]: B491C2C12CA: uid=987 from=<paheko>
Nov 16 03:56:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:56:01 rodinux postfix/qmgr[1808]: B491C2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:56:01 rodinux postfix/pipe[111135]: B491C2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:56:06 rodinux postfix/pipe[111135]: B96792C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:57:01 rodinux postfix/pickup[111544]: C915E2C12CA: uid=987 from=<paheko>
Nov 16 03:57:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:57:01 rodinux postfix/qmgr[1808]: C915E2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:57:01 rodinux postfix/pipe[111135]: C915E2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:57:06 rodinux postfix/pipe[111135]: CE2E02C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:58:01 rodinux postfix/pickup[111544]: DD9432C12CA: uid=987 from=<paheko>
Nov 16 03:58:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:58:01 rodinux postfix/qmgr[1808]: DD9432C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:58:01 rodinux postfix/pipe[111135]: DD9432C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:58:06 rodinux postfix/pipe[111135]: E2AB82C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:59:01 rodinux postfix/pickup[111544]: F19F42C12CA: uid=987 from=<paheko>
Nov 16 03:59:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 03:59:01 rodinux postfix/qmgr[1808]: F19F42C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 03:59:02 rodinux postfix/pipe[111135]: F19F42C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 03:59:07 rodinux postfix/pipe[111135]: 028602C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:00:02 rodinux postfix/pickup[111544]: 0F95B2C12CA: uid=987 from=<paheko>
Nov 16 04:00:02 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:00:02 rodinux postfix/qmgr[1808]: 0F95B2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:00:02 rodinux postfix/pipe[111135]: 0F95B2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:00:07 rodinux postfix/pipe[111135]: 137DF2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:01:01 rodinux postfix/pickup[111544]: 71B6E2C12CA: uid=987 from=<paheko>
Nov 16 04:01:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:01:01 rodinux postfix/qmgr[1808]: 71B6E2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:01:01 rodinux postfix/pipe[111135]: 71B6E2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:01:06 rodinux postfix/pipe[111135]: 7656F2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:02:01 rodinux postfix/pickup[111544]: 86FC02C12CA: uid=987 from=<paheko>
Nov 16 04:02:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:02:01 rodinux postfix/qmgr[1808]: 86FC02C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:02:01 rodinux postfix/pipe[111135]: 86FC02C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:02:06 rodinux postfix/pipe[111135]: 8B00D2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:03:01 rodinux postfix/pickup[111544]: 9DE1F2C12CA: uid=987 from=<paheko>
Nov 16 04:03:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:03:01 rodinux postfix/qmgr[1808]: 9DE1F2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:03:01 rodinux postfix/pipe[111135]: 9DE1F2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:03:06 rodinux postfix/pipe[111135]: A2EE12C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:04:01 rodinux postfix/pickup[112977]: B3BAC2C12CA: uid=987 from=<paheko>
Nov 16 04:04:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:04:01 rodinux postfix/qmgr[1808]: B3BAC2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:04:01 rodinux postfix/pipe[111135]: B3BAC2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.02/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:04:06 rodinux postfix/pipe[111135]: B8CA12C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:05:01 rodinux postfix/pickup[112977]: C58262C12CA: uid=987 from=<paheko>
Nov 16 04:05:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:05:01 rodinux postfix/qmgr[1808]: C58262C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:05:01 rodinux postfix/pipe[111135]: C58262C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:05:06 rodinux postfix/pipe[111135]: C98BE2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:06:01 rodinux postfix/pickup[112977]: 2DE902C12CA: uid=987 from=<paheko>
Nov 16 04:06:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:06:01 rodinux postfix/qmgr[1808]: 2DE902C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:06:01 rodinux postfix/pipe[111135]: 2DE902C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:06:06 rodinux postfix/pipe[111135]: 321D42C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:07:01 rodinux postfix/pickup[112977]: 4145C2C12CA: uid=987 from=<paheko>
Nov 16 04:07:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:07:01 rodinux postfix/qmgr[1808]: 4145C2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:07:01 rodinux postfix/pipe[111135]: 4145C2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:07:06 rodinux postfix/pipe[111135]: 463A92C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:08:01 rodinux postfix/pickup[112977]: 566CC2C12CA: uid=987 from=<paheko>
Nov 16 04:08:01 rodinux postsrsd[111148]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:08:01 rodinux postfix/qmgr[1808]: 566CC2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:08:01 rodinux postfix/pipe[111135]: 566CC2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:08:06 rodinux postfix/pipe[111135]: 5E1282C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0.02/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:09:01 rodinux postfix/pickup[112977]: 6A97E2C12CA: uid=987 from=<paheko>
Nov 16 04:09:01 rodinux postsrsd[113150]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:09:01 rodinux postfix/qmgr[1808]: 6A97E2C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:09:01 rodinux postfix/pipe[111135]: 6A97E2C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:09:06 rodinux postfix/pipe[111135]: 6F62E2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:10:01 rodinux postfix/pickup[112977]: 7CFE12C12CA: uid=987 from=<paheko>
Nov 16 04:10:01 rodinux postsrsd[113150]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:10:01 rodinux postfix/qmgr[1808]: 7CFE12C12CA: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 04:10:01 rodinux postfix/pipe[111135]: 7CFE12C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:10:06 rodinux postfix/pipe[111135]: 818812C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:11:01 rodinux postfix/pickup[112977]: EA10C2C14EE: uid=987 from=<paheko>
Nov 16 04:11:01 rodinux postsrsd[113150]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:11:01 rodinux postfix/qmgr[1808]: EA10C2C14EE: from=<paheko@rodinux.fr>, size=16211, nrcpt=1 (queue active)
Nov 16 04:11:01 rodinux postfix/pickup[112977]: EE2F32C12CA: uid=987 from=<paheko>
Nov 16 04:11:01 rodinux postsrsd[113150]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 04:11:01 rodinux postfix/qmgr[1808]: EE2F32C12CA: from=<paheko@rodinux.fr>, size=590, nrcpt=1 (queue active)
Nov 16 04:11:01 rodinux postfix/pipe[111135]: EE2F32C12CA: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 04:11:06 rodinux postfix/pipe[111135]: F244E2C148A: to=<paheko@rodinux.fr>, relay=dovecot, delay=4.9, delays=0/4.9/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:53:01 rodinux postfix/smtpd[131419]: Anonymous TLS connection established from localhost[::1] to paheko.rodinux.fr: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits) key-exchange X25519 server-signature RSA-PSS (3072 bits) server-digest SHA256
Nov 16 13:53:01 rodinux postfix/smtpd[131419]: NOQUEUE: reject: RCPT from localhost[::1]: 553 5.7.1 <contact@ecranvillage.net>: Sender address rejected: not owned by user paheko; from=<contact@ecranvillage.net> to=<rodolphe@linux07.fr> proto=ESMTP helo=<paheko.rodinux.fr>
Nov 16 13:53:01 rodinux postfix/pickup[124850]: 8EEFB2C12B4: uid=987 from=<paheko>
Nov 16 13:53:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:53:01 rodinux postfix/qmgr[1808]: 8EEFB2C12B4: from=<paheko@rodinux.fr>, size=1318, nrcpt=1 (queue active)
Nov 16 13:53:01 rodinux postfix/pipe[131719]: 8EEFB2C12B4: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.04, delays=0.02/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:53:06 rodinux postfix/pipe[131719]: 9618F2C14EE: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:54:01 rodinux postfix/pickup[131759]: A264B2C17D7: uid=987 from=<paheko>
Nov 16 13:54:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:54:01 rodinux postfix/qmgr[1808]: A264B2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:54:01 rodinux postfix/pipe[131719]: A264B2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.02/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:54:06 rodinux postfix/pipe[131719]: A6A3B2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:55:01 rodinux postfix/pickup[131759]: B503F2C17D7: uid=987 from=<paheko>
Nov 16 13:55:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:55:01 rodinux postfix/qmgr[1808]: B503F2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:55:01 rodinux postfix/pipe[131719]: B503F2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.02/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:55:06 rodinux postfix/pipe[131719]: B905E2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:56:01 rodinux postfix/pickup[131759]: 16EC62C17D7: uid=987 from=<paheko>
Nov 16 13:56:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:56:01 rodinux postfix/qmgr[1808]: 16EC62C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:56:01 rodinux postfix/pipe[131719]: 16EC62C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:56:06 rodinux postfix/pipe[131719]: 1B3F72C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:57:01 rodinux postfix/pickup[131759]: 2BD3A2C17D7: uid=987 from=<paheko>
Nov 16 13:57:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:57:01 rodinux postfix/qmgr[1808]: 2BD3A2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:57:01 rodinux postfix/pipe[131719]: 2BD3A2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:57:06 rodinux postfix/pipe[131719]: 2FF2B2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:58:01 rodinux postfix/pickup[131759]: 3F4322C17D7: uid=987 from=<paheko>
Nov 16 13:58:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:58:01 rodinux postfix/qmgr[1808]: 3F4322C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:58:01 rodinux postfix/pipe[131719]: 3F4322C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:58:06 rodinux postfix/pipe[131719]: 43B6F2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:59:01 rodinux postfix/pickup[131759]: 543D02C17D7: uid=987 from=<paheko>
Nov 16 13:59:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 13:59:01 rodinux postfix/qmgr[1808]: 543D02C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 13:59:01 rodinux postfix/pipe[131719]: 543D02C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 13:59:06 rodinux postfix/pipe[131719]: 5952D2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:00:01 rodinux postfix/pickup[131759]: 6696C2C17D7: uid=987 from=<paheko>
Nov 16 14:00:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:00:01 rodinux postfix/qmgr[1808]: 6696C2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:00:01 rodinux postfix/pipe[131719]: 6696C2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:00:06 rodinux postfix/pipe[131719]: 6AA8D2C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:01:01 rodinux postfix/pickup[131759]: C55E02C17D7: uid=987 from=<paheko>
Nov 16 14:01:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:01:01 rodinux postfix/qmgr[1808]: C55E02C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:01:01 rodinux postfix/pipe[131719]: C55E02C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:01:06 rodinux postfix/pipe[131719]: C9F752C17DC: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:02:01 rodinux postfix/pickup[131759]: D86212C17D7: uid=987 from=<paheko>
Nov 16 14:02:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:02:01 rodinux postfix/qmgr[1808]: D86212C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:02:01 rodinux postfix/pipe[131719]: D86212C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:02:06 rodinux postfix/pipe[131719]: DC9AB2C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:03:02 rodinux postfix/smtpd[132062]: Anonymous TLS connection established from localhost[::1] to paheko.rodinux.fr: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits) key-exchange X25519 server-signature RSA-PSS (3072 bits) server-digest SHA256
Nov 16 14:03:02 rodinux postfix/smtpd[132062]: NOQUEUE: reject: RCPT from localhost[::1]: 553 5.7.1 <contact@ecranvillage.net>: Sender address rejected: not owned by user paheko; from=<contact@ecranvillage.net> to=<rodolphe@linux07.fr> proto=ESMTP helo=<paheko.rodinux.fr>
Nov 16 14:03:02 rodinux postfix/pickup[131759]: 1455C2C17D7: uid=987 from=<paheko>
Nov 16 14:03:02 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:03:02 rodinux postfix/qmgr[1808]: 1455C2C17D7: from=<paheko@rodinux.fr>, size=1318, nrcpt=1 (queue active)
Nov 16 14:03:02 rodinux postfix/pipe[131719]: 1455C2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:03:07 rodinux postfix/pipe[131719]: 18EDE2C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:04:01 rodinux postfix/pickup[131759]: 27C302C17D7: uid=987 from=<paheko>
Nov 16 14:04:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:04:01 rodinux postfix/qmgr[1808]: 27C302C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:04:01 rodinux postfix/pipe[131719]: 27C302C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:04:06 rodinux postfix/pipe[131719]: 2C0872C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:05:01 rodinux postfix/pickup[131759]: 3A2952C17D7: uid=987 from=<paheko>
Nov 16 14:05:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:05:01 rodinux postfix/qmgr[1808]: 3A2952C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:05:01 rodinux postfix/pipe[131719]: 3A2952C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:05:06 rodinux postfix/pipe[131719]: 3E5272C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:06:01 rodinux postfix/pickup[131759]: 903CB2C17D7: uid=987 from=<paheko>
Nov 16 14:06:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:06:01 rodinux postfix/qmgr[1808]: 903CB2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:06:01 rodinux postfix/pipe[131719]: 903CB2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:06:06 rodinux postfix/pipe[131719]: 94D312C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:07:01 rodinux postfix/pickup[131759]: A55F32C17D7: uid=987 from=<paheko>
Nov 16 14:07:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:07:01 rodinux postfix/qmgr[1808]: A55F32C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:07:01 rodinux postfix/pipe[131719]: A55F32C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:07:06 rodinux postfix/pipe[131719]: A977A2C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:07:49 rodinux postfix/smtpd[132366]: Anonymous TLS connection established from localhost[::1] to paheko.rodinux.fr: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits) key-exchange X25519 server-signature RSA-PSS (3072 bits) server-digest SHA256
Nov 16 14:07:49 rodinux postfix/smtpd[132366]: 202CD2C1A40: client=localhost[::1], sasl_method=LOGIN, sasl_username=paheko
Nov 16 14:07:49 rodinux postsrsd[131717]: srs_forward: <paheko@paheko.rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:07:54 rodinux postfix/qmgr[1808]: 202CD2C1A40: from=<paheko@paheko.rodinux.fr>, size=16441, nrcpt=1 (queue active)
Nov 16 14:08:01 rodinux postfix/pickup[131759]: B84DD2C17D7: uid=987 from=<paheko>
Nov 16 14:08:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:08:01 rodinux postfix/qmgr[1808]: B84DD2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:08:01 rodinux postfix/pipe[131719]: B84DD2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:08:06 rodinux postfix/pipe[131719]: BC5822C1B21: to=<paheko@rodinux.fr>, relay=dovecot, delay=4.6, delays=0/4.6/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:09:01 rodinux postfix/pickup[131759]: CAA1D2C1B24: uid=987 from=<paheko>
Nov 16 14:09:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:09:01 rodinux postfix/qmgr[1808]: CAA1D2C1B24: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:09:01 rodinux postfix/pipe[131719]: CAA1D2C1B24: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:09:06 rodinux postfix/pipe[131719]: CEDC12C1B30: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:10:01 rodinux postfix/pickup[131759]: DCA872C10CB: uid=987 from=<paheko>
Nov 16 14:10:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:10:01 rodinux postfix/qmgr[1808]: DCA872C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:10:01 rodinux postfix/pipe[131719]: DCA872C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:10:06 rodinux postfix/pipe[131719]: E0F082C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:11:01 rodinux postfix/pickup[131759]: 458872C10CB: uid=987 from=<paheko>
Nov 16 14:11:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:11:01 rodinux postfix/qmgr[1808]: 458872C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:11:01 rodinux postfix/pipe[131719]: 458872C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:11:06 rodinux postfix/pipe[131719]: 49DF52C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:12:01 rodinux postfix/pickup[131759]: 5AC9E2C10CB: uid=987 from=<paheko>
Nov 16 14:12:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:12:01 rodinux postfix/qmgr[1808]: 5AC9E2C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:12:01 rodinux postfix/pipe[131719]: 5AC9E2C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:12:06 rodinux postfix/pipe[131719]: 5ED962C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:13:01 rodinux postfix/pickup[131759]: 6F9C82C10CB: uid=987 from=<paheko>
Nov 16 14:13:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:13:01 rodinux postfix/qmgr[1808]: 6F9C82C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:13:01 rodinux postfix/pipe[131719]: 6F9C82C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:13:05 rodinux postfix/qmgr[1808]: 202CD2C1A40: from=<paheko@paheko.rodinux.fr>, size=16441, nrcpt=1 (queue active)
Nov 16 14:13:06 rodinux postfix/pipe[131719]: 73FCD2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:14:01 rodinux postfix/pickup[131759]: 84AE32C10CB: uid=987 from=<paheko>
Nov 16 14:14:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:14:01 rodinux postfix/qmgr[1808]: 84AE32C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:14:01 rodinux postfix/pipe[131719]: 84AE32C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:14:06 rodinux postfix/pipe[131719]: 891872C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:15:01 rodinux postfix/smtpd[132822]: Anonymous TLS connection established from localhost[::1] to paheko.rodinux.fr: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits) key-exchange X25519 server-signature RSA-PSS (3072 bits) server-digest SHA256
Nov 16 14:15:01 rodinux postfix/smtpd[132822]: A91C72C17D7: client=localhost[::1], sasl_method=LOGIN, sasl_username=paheko
Nov 16 14:15:01 rodinux postsrsd[131717]: srs_forward: <paheko@paheko.rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:15:01 rodinux postfix/qmgr[1808]: A91C72C17D7: from=<paheko@paheko.rodinux.fr>, size=16288, nrcpt=1 (queue active)
Nov 16 14:15:01 rodinux postfix/smtpd[132822]: Anonymous TLS connection established from localhost[::1] to paheko.rodinux.fr: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits) key-exchange X25519 server-signature RSA-PSS (3072 bits) server-digest SHA256
Nov 16 14:15:01 rodinux postfix/smtpd[132822]: NOQUEUE: reject: RCPT from localhost[::1]: 550 5.1.1 <paheko@paheko.rodinux.fr>: Recipient address rejected: User unknown in virtual mailbox table; from=<paheko@paheko.rodinux.fr> to=<paheko@paheko.rodinux.fr> proto=ESMTP helo=<paheko.rodinux.fr>
Nov 16 14:15:01 rodinux postfix/pickup[131759]: EF6FD2C10CB: uid=987 from=<paheko>
Nov 16 14:15:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:15:01 rodinux postfix/qmgr[1808]: EF6FD2C10CB: from=<paheko@rodinux.fr>, size=1334, nrcpt=1 (queue active)
Nov 16 14:15:01 rodinux postfix/pipe[131719]: EF6FD2C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:15:07 rodinux postfix/pipe[131719]: F36802C1B21: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:16:02 rodinux postfix/pickup[131759]: 105B72C10CB: uid=987 from=<paheko>
Nov 16 14:16:02 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:16:02 rodinux postfix/qmgr[1808]: 105B72C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:16:02 rodinux postfix/pipe[131719]: 105B72C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:16:07 rodinux postfix/pipe[131719]: 14EA02C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:17:01 rodinux postfix/pickup[131759]: 20C982C10CB: uid=987 from=<paheko>
Nov 16 14:17:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:17:01 rodinux postfix/qmgr[1808]: 20C982C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:17:01 rodinux postfix/pipe[131719]: 20C982C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:17:06 rodinux postfix/pipe[131719]: 24DAC2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.03, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:18:01 rodinux postfix/pickup[131759]: 3450B2C10CB: uid=987 from=<paheko>
Nov 16 14:18:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:18:01 rodinux postfix/qmgr[1808]: 3450B2C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:18:01 rodinux postfix/pipe[131719]: 3450B2C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:18:05 rodinux postfix/postqueue[132894]: input attribute value: paheko@rodinux.fr
Nov 16 14:18:05 rodinux postfix/postqueue[132894]: input attribute value: paheko@paheko.rodinux.fr
Nov 16 14:18:06 rodinux postfix/pipe[131719]: 3A99A2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:19:01 rodinux postfix/pickup[131759]: 48DE42C10CB: uid=987 from=<paheko>
Nov 16 14:19:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:19:01 rodinux postfix/qmgr[1808]: 48DE42C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:19:01 rodinux postfix/pipe[131719]: 48DE42C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:19:06 rodinux postfix/pipe[131719]: 4E18C2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:20:01 rodinux postfix/pickup[131759]: 5B45B2C10CB: uid=987 from=<paheko>
Nov 16 14:20:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:20:01 rodinux postfix/qmgr[1808]: 5B45B2C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:20:01 rodinux postfix/pipe[131719]: 5B45B2C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:20:06 rodinux postfix/pipe[131719]: 5F50F2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:21:01 rodinux postfix/pickup[131759]: B68C22C10CB: uid=987 from=<paheko>
Nov 16 14:21:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:21:01 rodinux postfix/qmgr[1808]: B68C22C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:21:01 rodinux postfix/pipe[131719]: B68C22C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:21:06 rodinux postfix/pipe[131719]: BB5D82C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:21:34 rodinux postfix/qmgr[1808]: 202CD2C1A40: from=<paheko@paheko.rodinux.fr>, size=16441, nrcpt=1 (queue active)
Nov 16 14:22:01 rodinux postfix/pickup[131759]: C9CA32C10CB: uid=987 from=<paheko>
Nov 16 14:22:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:22:01 rodinux postfix/qmgr[1808]: C9CA32C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:22:01 rodinux postfix/pipe[131719]: C9CA32C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:22:06 rodinux postfix/pipe[131719]: CE7092C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:23:01 rodinux postfix/pickup[131759]: E06242C10CB: uid=987 from=<paheko>
Nov 16 14:23:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:23:01 rodinux postfix/qmgr[1808]: E06242C10CB: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:23:01 rodinux postfix/pipe[131719]: E06242C10CB: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:23:06 rodinux postfix/pipe[131719]: E4AAC2C17D7: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:24:02 rodinux postfix/pickup[131759]: F41E72C17D7: uid=987 from=<paheko>
Nov 16 14:24:02 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:24:02 rodinux postfix/qmgr[1808]: F41E72C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:24:02 rodinux postfix/pipe[131719]: F41E72C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:24:07 rodinux postfix/pipe[131719]: 04D872C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:25:02 rodinux postfix/pickup[131759]: 123472C17D7: uid=987 from=<paheko>
Nov 16 14:25:02 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:25:02 rodinux postfix/qmgr[1808]: 123472C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:25:02 rodinux postfix/pipe[131719]: 123472C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.02, delays=0.01/0/0/0.01, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:25:07 rodinux postfix/pipe[131719]: 163D52C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:26:01 rodinux postfix/pickup[131759]: 786FB2C17D7: uid=987 from=<paheko>
Nov 16 14:26:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:26:01 rodinux postfix/qmgr[1808]: 786FB2C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:26:01 rodinux postfix/pipe[131719]: 786FB2C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.02/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:26:06 rodinux postfix/pipe[131719]: 7DCCD2C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:27:01 rodinux postfix/pickup[131759]: 8D4862C17D7: uid=987 from=<paheko>
Nov 16 14:27:01 rodinux postsrsd[131717]: srs_forward: <paheko@rodinux.fr> not rewritten: Domain excluded by policy
Nov 16 14:27:01 rodinux postfix/qmgr[1808]: 8D4862C17D7: from=<paheko@rodinux.fr>, size=600, nrcpt=1 (queue active)
Nov 16 14:27:01 rodinux postfix/pipe[131719]: 8D4862C17D7: to=<paheko@rodinux.fr>, orig_to=<paheko>, relay=dovecot, delay=0.03, delays=0.01/0/0/0.02, dsn=5.1.1, status=bounced (user unknown)
Nov 16 14:27:06 rodinux postfix/pipe[131719]: 9282B2C1A40: to=<paheko@rodinux.fr>, relay=dovecot, delay=5, delays=0/5/0/0.02, dsn=5.1.1, status=bounced (user unknown)
```